### PR TITLE
Add foreman_statistics repo

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Process `foreman_statistics`

### CODE REVIEW

File: config/repos.yaml [hunk 1]
```diff
@@ -93,6 +93,13 @@ theforeman/foreman-selinux:
   redmine_required: true
 theforeman/foreman_setup:
   redmine: foreman_setup
+theforeman/foreman_statistics:
+  redmine: statistics
+  redmine_required: true
+  branch_labels:
+    .+-stable: Stable branch
+  path_labels:
+    webpack: UI
```
Comment (ekohl) [inline config/repos.yaml:102]: Note that the processor doesn't create these labels. It's up to you to create them.

Comment (ezr-ondrej) [reply config/repos.yaml:102]: Thanks for the heads up, noted, but I suspected that being the case, so I got them created: https://github.com/theforeman/foreman_statistics/labels

**Review by tbrisker — APPROVED** [review]
thanks @ezr-ondrej !

**Comment by ekohl** [comment]
@tbrisker did you also deploy this?

**Comment by tbrisker** [comment]
@ekohl no, it isn't automatically deployed?

**Comment by ekohl** [comment]
No

**Comment by tbrisker** [comment]
@ekohl I think the readme or wiki needs to be updated with how to deploy this, the wiki currently mentions pushing to openshift which I think hasn't been correct in years, and the readme has no information. Alternatively we should set it up so changes are deployed automatically after merge and then noone needs to worry about it.

**Pull request merged**